Empty dTag transfer fix (#303)
desmos-labs/desmos: CHANGELOG.md

# Version 0.13.0
## Changes
- Removed the relationship event attribute key prefix (#300)
- Removed the user_block event attribute key prefix (#291)

> - Fixed the possibility to request a transfer for an empty dTag (#292)

Reviewer: ```suggestion
- Fixed the possibility of requesting a transfer of an empty DTag (#292)
```
